niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/config/InstallPermissionOnlyConfiguration.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.certifications.niap.permissions.config;
+
+import android.app.Activity;
+
+import com.android.certifications.niap.permissions.BasePermissionTester;
+import com.android.certifications.niap.permissions.DevicePolicyPermissionTester;
+import com.android.certifications.niap.permissions.InstallPermissionTester;
+import com.android.certifications.niap.permissions.R;
+import com.android.certifications.niap.permissions.utils.PermissionUtils;
+import com.android.certifications.niap.permissions.utils.ReflectionUtils;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Configuration to verify the Google Play Services {GMS} client side libraries properly guard their
```

Comment:
Can you update this Javadoc to reflect the summary of this configuration class?

Reply:
Thanks will update it

Reply:
Updated comment sounds good, thanks!